services/models.py

```diff
@@ -17,6 +17,7 @@ class Service(models.Model):
     url = models.CharField(max_length=255, null=False, blank=False)
     token = models.CharField(max_length=40, null=False, blank=False)
     up = models.BooleanField(default=False)
+    last_up_at = models.DateTimeField(null=True)
```

Comment:
Do we need this field? Can we not just do a `Status.objects.filter(service=service, up=True).order_by(-'created_at').first()`? We can even have that as a method/property on the `Service` model.